### ISSUE TITLE
Indicate when BAMs are missing

### ISSUE THREAD

**Issue by ryan-williams** [hammerlab/cycledash#439]
I was confused by the absence of a pileup view here:

![image](https://cloud.githubusercontent.com/assets/465045/5943307/d9200a1a-a6eb-11e4-9198-becc83b6a1db.png)

Can we do any/all of or something similar to:
* have the metadata show that no BAMs were added
* disable the "open pileup viewer" link
* allow a button that allows a user to attach the BAMs for a run
?

Or maybe something else?

**Comment by danvk** [comment]
Indicating that BAMs are missing & letting a user attach them post-submission sounds reasonable. I think there's still some (marginal) value in having the pileup viewer w/o BAMs, especially if it shows the VCF track. It can put the call in context (e.g. if it's in a highly repetitive region).

**Comment by ihodes** [comment]
Makes sense. Worth noting that data *is* shown if there *are* BAMs, and that the API does support adding them to existing runs with a PUT request. Exposing this via the UI will be simple.

**Comment by ryan-williams** [comment]
great, yea agreed that just having some UI indication of why there's no pileup would solve this. if there was a way to indicate that near the biodalliance VCF track, that would be a great added bonus

**Comment by ryan-williams** [comment]
bumping this, can we indicate presence/absence of BAMs on the `/runs` page?

**Comment by ryan-williams** [comment]
ah wait, that's what the "tumor" and "normal" tags indicate, right?

Just tried on [this run](http://cycledash.demeter.hpc.mssm.edu/runs/153/examine?query=ORDER+BY+contig%2C+position) and got "Couldn't access BAI" errors:

![](https://s3.amazonaws.com/f.cl.ly/items/2u1e2f0F0k2X3E1I2m1X/Screen%20Shot%202015-03-17%20at%202.50.30%20PM.png)

Can we figure that out earlier?

I just want to know which runs I can actually see pileups for without having to try a bunch.

**Comment by danvk** [comment]
Yep, that's what the tumor/normal badges indicate. If the BAM file is `/foo/bar.bam`, then the BAI file needs to be `/foo/bar.bam.bai`. If you rename your BAI file to match that pattern, it should work.

**Comment by ryan-williams** [comment]
Can we make cycledash do something more graceful here?

1. indicate that the BAMs, despite existing in some sense, will actually not load if I attempt to see pileups.
2. have CD create a `.bai` on the fly in a situation like this.

In this case these aren't my BAMs and I don't know where they are, so it would be nice if the app helped me out a little bit. 

Orthogonally, displaying the paths to the BAMs somewhere would be useful.

**Comment by danvk** [comment]
I think it displays the path to the BAMs on the /runs page.
